JS-exe/JS-5.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Grade Checker</title>
+</head>
+<body>
+    <form id="gradeForm">
+        <label for="grade">Grade:</label>
+        <input type="text" id="grade" name="grade" required>
+        <input type="submit" value="Check Grade" onclick="myFunction(); return false;">
+    </form>
+    
+    <script>
+        myFunction = () => {
```

Comment:
- Pay attention: seperating js code into js file is important
- The paragraph should be replaced, not added 
- You should print the char as well